main.go

```diff
@@ -120,9 +120,10 @@ func main() {
 	}
 }
 
-func isDebug() bool {
-	return os.Getenv("DEBUG") == "true"
-}
+// unused function
+//func isDebug() bool {
+//	return os.Getenv("DEBUG") == "true"
+//}
 
```

Comment:
Instead of commenting, lets remove this function if not in use
```suggestion

```

Reply:
done!